src/main.cpp

```diff
@@ -2405,7 +2408,7 @@ void static UpdateTip(CBlockIndex *pindexNew) {
 }
 
 /** Disconnect chainActive's tip. */
-bool static DisconnectTip(CValidationState &state) {
+bool static DisconnectTip(CValidationState &state, bool fBare = false) {
```

Comment:
Is the default for `fBare` supposed to be here or in the header?

Reply:
There is no entry for this function in the header, as it's internal. This is the declaration, so the default goes here.

Reply:
C++, Y U so inconsistent? (Does not block ;-) )